independent-projects/arc/processor/src/main/java/io/quarkus/arc/processor/BeanDeployment.java

```diff
@@ -355,12 +362,49 @@ void init() {
 
     static Map<DotName, ClassInfo> findInterceptorBindings(IndexView index) {
         Map<DotName, ClassInfo> bindings = new HashMap<>();
+        // Note: doesn't use AnnotationStore, this will operate on classes without applying annotation transformers
         for (AnnotationInstance binding : index.getAnnotations(DotNames.INTERCEPTOR_BINDING)) {
             bindings.put(binding.target().asClass().name(), binding.target().asClass());
         }
         return bindings;
     }
 
+    Map<DotName, Set<AnnotationInstance>> findTransitiveInterceptorBindigs(Collection<DotName> initialBindings, IndexView index,
+            Map<DotName, Set<AnnotationInstance>> result, AnnotationStore store) {
```

Comment:
There is no need to pass the `AnnotationStore` - it's a field of `BeanDeployment`.